connect.go

```diff
@@ -93,6 +98,7 @@ func getDefaultFetchOptions() FetchOpts {
 		ErrHandler:               DefaultConsumerErrHandler,
 		StartConsumeFromSequence: 1,
 		LastMessages:             -1,
+		Prefetch:                 true,
```

Comment:
the default should be false

Reply:
Okay, updated prefetch to be false by default.